### PR TITLE
Added language icons modal on Search page

### PR DESCRIPTION
**Pull request by Anahkiasen**
### Added
- Added Editorconfig file to ensure consistent code style
- Added language icons modal on Search page

### Fixed
- Fixed some `imagemin` dependencies not being declared at the top level

---

https://github.com/cultuurnet/udb3-angular/assets/1321596/8d40a76d-3087-45b5-98e7-c15dce23f7e9

Requires https://github.com/cultuurnet/udb3-angular-app/pull/211

Ticket: https://jira.publiq.be/browse/III-5982

### CODE REVIEW

**Comment by Anahkiasen** [comment]
Forgot to conditionally hide depending on app language

**Review by simon-debruijn — APPROVED** [review]

File: src/entry/labelling/offer-languages-modal.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<div class="modal-body offer-languages-modal">
+  <h2>Taaliconen toevoegen</h2>
```
Comment (brampauwelyn) [inline src/entry/labelling/offer-languages-modal.html:2]: Could probably also be moved to the translations file?

File: src/entry/labelling/offer-languages-modal.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <div class="alert alert-info" role="alert">
+    Aan de hand van taaliconen geef je aan hoeveel kennis van het Nederlands vereist is om aan je aanbod te kunnen
```
Comment (brampauwelyn) [inline src/entry/labelling/offer-languages-modal.html:4]: Could probably also be moved to the translations file?

File: src/search/ui/search.html [hunk 3]
```diff
@@ -64,6 +64,9 @@
                     <button class="btn btn-default rv-action" ng-click="label()">
                         <i class="fa fa-tag"></i> <span translate-once="search.label"></span>
                     </button>
+                    <button class="btn btn-default rv-action" ng-click="addLanguageIcons()" ng-if="language == 'nl'">
```
Comment (brampauwelyn) [inline src/search/ui/search.html:67]: Why not use a strict comparison?
```suggestion
                    <button class="btn btn-default rv-action" ng-click="addLanguageIcons()" ng-if="language === 'nl'">
```

**Review by brampauwelyn — APPROVED** [review]

**Comment by Anahkiasen** [comment]
@brampauwelyn Sorry for the translations they were kept intentionally hardcoded as per the ticket since this is a dutch-specific feature:

>  Language icons are only applicable in the Flemish region of Belgium, so it doesn't make sense to provide this functionality for users using the FR or DE interface, or to provide FR or DE copy for it.

**Comment by brampauwelyn** [comment]
> @brampauwelyn Sorry for the translations they were kept intentionally hardcoded as per the ticket since this is a dutch-specific feature:
> 
> > Language icons are only applicable in the Flemish region of Belgium, so it doesn't make sense to provide this functionality for users using the FR or DE interface, or to provide FR or DE copy for it.

Ah yes true 👍

**Pull request merged**